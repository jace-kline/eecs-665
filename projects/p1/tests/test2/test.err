FATAL [1,52]: Illegal character \
FATAL [1,55]: Illegal character .
FATAL [1,56]: Illegal character .
FATAL [1,57]: Illegal character .
FATAL [1,77]: Illegal character ?
FATAL [1,78]: Empty character literal
flex scanner jammed
