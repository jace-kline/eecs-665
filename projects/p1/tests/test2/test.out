ID:his [1,3]
ID:is [1,7]
ID:a [1,10]
ID:test [1,12]
ID:of [1,17]
ID:newline [1,20]
ID:literal [1,28]
ID:within [1,36]
ID:a [1,43]
ID:string [1,45]
ID:n [1,53]
ID:did [1,59]
ID:I [1,63]
ID:get [1,65]
ID:an [1,69]
ID:error [1,72]
ID:his [1,82]
ID:is [1,86]
ID:a [1,89]
ID:test [1,91]
ID:of [1,96]
ID:newline [1,99]
ID:key [1,107]
ID:within [1,111]
ID:a [1,118]
ID:string [1,120]
